package.json

```diff
@@ -9,7 +9,7 @@
   },
   "scripts": {
     "start": "npm run test:story",
-    "build": "npm run clean && gulp build-ts && gulp build-modules",
+    "build": "npm run clean && gulp --gulpfile build/gulpfile.js build-ts && gulp --gulpfile build/gulpfile.js build-modules",
```

Comment:
why don't you create a default gulp task that runs both task?

Reply:
Merged into a single task.